src/io/flutter/view/InspectorPerfTab.java

```diff
@@ -56,7 +56,13 @@ public void setVisibleToUser(boolean visible) {
       app.getPerfService().resumePolling();
     }
     else {
-      app.getPerfService().pausePolling();
+      // TODO(terry): Currently, don't pause polling so all profiler views
```

Comment:
I would instead tweak the perf service class so it tracks the # of consumers requesting polling instead of just supporting resumePolling and pausePolling. That way, your perf tab won't be broken by code in a different tab that chooses to stop listening for events when the tab is closed. I expect this class was written the way it was for simplicity but now that we have multiple consumers it is time to make it more robust.
E.g. you could just add
app.getPerfService.addPollingListener(listener);
app.getPerfService.removePollingListener(listener);
or just app.getPerfService.addPollingClient()
and app.getPerfService.removePollingClient() if you want to keep it simple
with just a count in PerfService tracking whether there are any clients.

Reply:
Good idea.  If memory profiling is enabled we're always collecting @ time 0:0(regardless of tab being selected).